tests/integration/templates/Towerfile.j2

```diff
@@ -0,0 +1,8 @@
+[app]
```

Comment:
this ends with `.j2` as if it's a jinja template, and it sort of is, but in this case I'm quite literally doing a find and replace for these specific variables: https://github.com/tower/tower-cli/pull/86/commits/6f04369223e9dd3739f6819900edea03ac5014cf#diff-365b24b9adc19a6d5ff90a9eb58bbc764d70284234d4dcc97bd35bc5d5a7b5f3R43